app/facades/order/order_facade.rb

```diff
@@ -0,0 +1,31 @@
+class Order
+  class OrderFacade
+    def initialize; end
```

Comment:
You can just remove it, each class has `initialize` method by default since they're all inherited from `Object`